content/applications/sales/sales/send_quotations/deadline.rst

```diff
@@ -1,44 +1,57 @@
-===================================================
-Use quotations deadline to stimulate your customers
-===================================================
+==========================================
+Quotation deadlines to stimulate customers
+==========================================
 
-When sending quotations, it is important to set a deadline to encourage your customers to act.
-Indeed, this will stimulate them because they will be afraid of missing a good deal and it will also
-allow you to protect yourself in case you have to fulfill an order at a price that is no longer
-profitable for you.
+When sending quotations, it's important to set a deadline to encourage customers to act and confirm
+orders in a timely manner.
 
-Set a deadline
-==============
+Setting deadlines on quotations entices customers to finalize the process because they may fear
+that they'll miss out on a good deal. Quotation deadlines are also useful because they can be used
+as protection for a company, in case an order has to be fulfilled at a price that is no longer
+profitable for the business.
 
-With Odoo Sales, it is possible to instantly add an **Expiration Date** from the quotation or the
-sales order.
+Expiration date deadlines
+=========================
 
-.. image:: deadline/quotation_deadlines_1.png
+With Odoo *Sales*, it's possible to instantly add an expiration date in the :guilabel:`Expiration`
+field of a quotation or sales order.
+
+.. image:: deadline/quotation-deadlines-1.png
```

Comment:
Please update the image file names to follow the [image file naming convention](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files).